apps/modernization-ui/src/pages/addPatient/components/otherInfoFields/OtherInfoFields.tsx

```diff
@@ -28,15 +28,19 @@ export default function OtherInfoFields({ id, title, control, coded }: Props) {
                 <Grid row>
                     <Grid col={6}>
                         <Controller
+                            rules={{
+                                required: { value: true, message: 'This field is required' }
+                            }}
                             control={control}
-                            name="dateOfBirth"
-                            render={({ field: { onChange, value, name } }) => (
+                            name="dob"
```

Comment:
There may have been a few merge conflicts in this file resulting in changes being reverted.  The names of the form properties were aligned with the names expected by the API.  

The `render` property of the [Controller](https://react-hook-form.com/docs/usecontroller/controller) exposes two pretty useful objects `field` and `fieldState`.  

`field` is a render prop that provides state and functions to the component rendering the controlled form field.  The `name` from the `Controller` component is available here which prevents having to make sure that the `Controller` and component have the same name and id's.  

[fieldState](https://react-hook-form.com/docs/useform/getfieldstate) contains `error` and `invalid` properties that are set based on the `rules` from the `Controller`.  This means that the `render` property can provide the error state of the controlled form field without the need to pass down the errors from the `form`.  

`render={({ field: { onChange, value, name }, fieldState:{ error } }) => (` will contain any errors for this field only (if present).  There are a couple of examples of it being used [here](https://github.com/CDCgov/NEDSS-Modernization/blob/665b5074d024f534841898d2fef695add26bf429/apps/modernization-ui/src/pages/patient/profile/addresses/AddressEntryForm.tsx#L74)